juju/juju
# chore: upgrade to latest version of SQLair

Upgrade to the latest version of SQLair, it had been a while since this was last done. This change includes an extra error check for unsed structs in prepare statements.

## QA steps

Make sure all SQLair prepare statements in the codebase are run to check the new error check in SQLair does not catach any unused parameters.  This should be covered by the unit tests.
